caddy/caddy.go

```diff
@@ -412,6 +432,17 @@ func (f *FrankenPHPModule) Provision(ctx caddy.Context) error {
 		}
 	}
 
+	data := []byte(f.Root + strings.Join(f.SplitPath, ",") + time.Now().String())
+	hash := sha256.Sum256(data)
+	f.ModuleID = hex.EncodeToString(hash[:8])
+
+	if len(f.Workers) > 0 {
+		for _, w := range f.Workers {
+			w.ModuleID = f.ModuleID
+		}
+		moduleWorkers = append(moduleWorkers, f.Workers...)
+	}
```

Comment:
Generate some random hash. Can't do it before in php_server UnmarshalCaddyfile, because it has to be the same for every worker of this php_server directive.